src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -560,9 +639,13 @@ private static Collection<ScriptBindingProvider> LoadBindingProviders(ScriptHost
                 typeof(NotificationHubScriptBindingProvider),
                 typeof(SendGridScriptBindingProvider),
                 typeof(TwilioScriptBindingProvider),
-                typeof(BotFrameworkScriptBindingProvider)
-            };
+                typeof(BotFrameworkScriptBindingProvider),
```

Comment:
The primary goal of this review is to allow *new* extensions.  Removing the existing ones is a separate goal.

---
In reply to: [109000947](https://github.com/Azure/azure-webjobs-sdk-script/pull/1349#discussion_r109000947) [](ancestors = 109000947)